app/assets/stylesheets/application.scss

```diff
@@ -10,6 +10,23 @@
  * defined in the other CSS/SCSS files in this directory. It is generally better to create a new
  * file per style scope.
  *
- *= require_tree .
- *= require_self
  */
+ @import "bootstrap-sprockets";
+ @import "bootstrap";
+
+.page_header {
+  background: url(http://media-ranker.herokuapp.com/assets/owl-5e51d9e0d49c787f2ebd143e32cbf41c.jpg);
```

Comment:
This definitely works in this case, but for other projects, it is usually a good idea to refer to an image in your directory (like saving this image into your own folder) just in case anything happens to this 3rd party asset (ex: it gets deleted).